client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -33,9 +71,14 @@ export default function OnboardingEdit() {
 				</StepperWizard>
 				{ isFilledFormValue( siteVertical ) && (
 					<div className="onboarding-block__footer">
-						<Button className="onboarding-block__question-skip" isLink>
+						<Link
+							component={ LinkButton }
```

Comment:
Why not keep using the `<Button/>` and add `onClick` handler to navigate?
https://reacttraining.com/react-router/web/api/Hooks/usehistory

Reply:
That seems reasonable, I'm not opposed if it works well with the different routers. I had to use the hash router to get everything working alongside page.js.

`Link` seems to be the preferred ReactRouter navigational element, but it doesn't work well with our Button component. Another option would be to put all of the (copied) cruft here into its own module so we can share a simple API:

```tsx
import ButtonLink from '…'; 
<ButtonLink to="/fse">Go!</ButtonLink>
```

Feel free to explore what makes sense to you in this PR 👍

Reply:
Having a simple and 'router-like' API in one of our own modules is totally fine. 

By looking at the current implementation I see we normally render in DOM an `<a />` (coming from `react-router-dom` Link) but when `to` is undefined we render a `<button />` (coming from `@wordpress/components` Button)

I understand its on their roadmap to add an `useNavigation` hook but until then we could start using `useHistory` within some abstraction like this:

```
const Link = ( { to, onClick, ...props } ) => {
	const history = useHistory();
	return (
		<Button
			{ ...props }
			onClick={ e => {
                               history.push( to );
			      // + click handling from https://github.com/Automattic/wp-calypso/pull/37921/files#diff-5a85a4a3d01d8eee91208a69bb94bd83R29-R32
			} }
			disabled={ ! to }
		/>
	);
};
```

Reply:
If I'm not mistaken, everything is now using our wrapper around react-router `Link`, which uses `@wordpress/components` `Button` as it's rendered component. Nothing should be using react-router `Link` directly (except the wrapper).